### DISCUSSION TITLE
additional dataset

### DISCUSSION THREAD

**Discussion by 21pl** [plemeri/transparent-background#29] [Ideas]
First of all congratulations on this amazing work. I have been using this extensively on both single images and videos and I am always impressed by the results.
I do, however, want to point out an issue I see. Not really an issue but more of a want to enhance the results. With professional imagery where the foreground starts to fall out of focus the algorithm sometimes is unable to find this edge. This is understandable given the training data where everything has a sharp edge. Would it be worthwhile to preprocess some of the dataset with added blurs and gradient blurs to account for this type of imagery?

**Comment by plemeri** [comment]
Hi @21pl, Thank you for your attention to our work.

Before the discussion, can you share some of examples to explain the issue that you mentioned?
I would like to see first those professional imagery with out-of-focused foregrounds.

Thank you!
